docs/website/versioned_docs/version-3.0.0/command-reference/list-namespace.md

```diff
@@ -0,0 +1,50 @@
+---
+title: odo list namespace
+sidebar_position: 2
+---
+
+You can use `odo list namespace` to list all the available namespaces within the cluster. If you are on a Kubernetes cluster, this command will return a list of Namespace resources, but on an OpenShift cluster, it will return a list of Project resource.
```

Comment:
```suggestion
You can use `odo list namespace` to list all the available namespaces within the cluster. If you are on a Kubernetes cluster, this command will return a list of Namespace resources, but on an OpenShift cluster, it will return a list of Project resources.
```